dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl

```diff
@@ -113,4 +113,11 @@ namespace Microsoft.Windows.ApplicationModel.Resources
         static String TargetSize { get; };
         static String Theme { get; };
     }
-} // namespace Microsoft.Windows.ApplicationModel.Resources
+
+    [contract(MrtCoreContract, 1)]
```

Comment:
>1 [](http://example.com/codeflow?start=31&length=1)

Since this is a new api, we should bump this to 2.

Reply:
fixed